src/code42cli/cmds/watchlists.py

```diff
@@ -0,0 +1,252 @@
+import csv
+
+import click
+from pandas import DataFrame
+from py42.constants import WatchlistType
+from py42.exceptions import Py42NotFoundError
+from py42.exceptions import Py42WatchlistNotFound
+
+from code42cli.bulk import generate_template_cmd_factory
+from code42cli.bulk import run_bulk_process
+from code42cli.click_ext.groups import OrderedGroup
+from code42cli.click_ext.options import incompatible_with
+from code42cli.click_ext.types import AutoDecodedFile
+from code42cli.errors import Code42CLIError
+from code42cli.options import format_option
+from code42cli.options import sdk_options
+from code42cli.output_formats import DataFrameOutputFormatter
+
+
+@click.group(cls=OrderedGroup)
+@sdk_options(hidden=True)
+def watchlists(state):
+    """Manage watchlist user memberships."""
+    pass
+
+
+@watchlists.command("list")
+@format_option
+@sdk_options()
+def _list(state, format):
+    """List all watchlists."""
+    pages = state.sdk.watchlists.get_all()
+    dfs = (DataFrame(page["watchlists"]) for page in pages)
+    formatter = DataFrameOutputFormatter(format)
+    formatter.echo_formatted_dataframes(dfs)
+
+
+@watchlists.command()
+@click.option(
+    "--watchlist-id",
+    help="ID of the watchlist.",
+)
+@click.option(
+    "--watchlist-type",
+    type=click.Choice(WatchlistType.choices()),
+    help="Type of watchlist to list.",
+    cls=incompatible_with("watchlist_id"),
+)
+@format_option
+@sdk_options()
+def list_members(state, watchlist_type, watchlist_id, format):
```

Comment:
I feel like we may still want a `list_included_users` command (or some way to view that data, maybe even just a `--only-show-included-users` flag?) so that people managing adding/removing individual users will have a way to see only the individuals that can be managed. May be confusing if someone sees someone whose a member but isn't able to remove them? 


We'd also want to specify the difference between the two in that case